build.gradle

```diff
@@ -303,26 +306,58 @@ task coverage {
 task checkStyle {
 }
 
+ext {
+    guiTests = 'address/guitests/*'
+    guiUnitTests = 'address/guiunittests/*'
+    unitTests = 'address/unittests/*'
```

Comment:
Do you mean the case where we further sub-divide the unit tests? Then this should be extendable, unless the group we wish to test originates from various packages e.g. unittest/A and unittest/B under a fileRelatedTest task. In that case we have to use an array of filters instead.